courses/machine_learning/deepdive2/structured/solutions/3a_bqml_baseline_babyweight.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<ul><li>"Verify that you" -&gt; "Run the following cells to verify that"</li><li>The link&nbsp;"<a href="https://github.com/GoogleCloudPlatform/training-data-analyst/raw/e583c2c028f8a123e99b6f4954ceae24acd709d3/courses/machine_learning/deepdive2/structured/solutions/2_prepare_babyweight.ipynb" target="_blank">2_prepare_babyweight</a>&nbsp;" does not work</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done</p>